src/sentry/migrations/0188_backfill_changes.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 1.11.29 on 2021-04-22 01:26
+
+from django.db import migrations
+
+
+def backfill_incidents(apps, schema_editor):
+    Incident = apps.get_model("sentry", "Incident")
+    for incident in Incident.objects.all():
+        incident.update(subtitle=f"{incident.title} - some value")
```

Comment:
We should just update the model instance and call `save`. `update` isn't built into Django and it's possible a future version of Sentry might be missing the function.